client/client_test.go

```diff
@@ -97,6 +97,36 @@ func TestClient_Sign(t *testing.T) {
 	}
 }
 
+func TestClient_Encrypt(t *testing.T) {
```

Comment:
You are mixing CamelCase and snake_case. Just stick to camel case (Drop the `_`)

Reply:
Ahh got it! I think the other testing functions in the file followed that format but I'll change this!